### PR TITLE
Update Authorization install to use DIU for configs

### PR DESCRIPTION
**Pull request by ctediz**
Update to latest DIU

Add Get-AuthorizationFabricInstallerSecret tests

### CODE REVIEW

File: Fabric.Authorization.API/scripts/Install-Authorization.ps1 [hunk 1]
```diff
@@ -46,34 +46,41 @@ if(!(Test-IsRunAsAdministrator))
 
 $ErrorActionPreference = "Stop"
 
+# Grab install configs
 Write-DosMessage -Level "Information" -Message "Using install.config: $installConfigPath"
+$configStore = @{Type = "File"; Format = "XML"; Path = "$installConfigPath"}
 $installSettingsScope = "authorization"
```
Comment (hckenmiller) [inline Fabric.Authorization.API/scripts/Install-Authorization.ps1]: maybe having this as $authSettingsScope

File: Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1 [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                $decryptedSecret | Should -eq $fabricInstallerSecret
+                Assert-MockCalled Unprotect-DosInstallerSecret -Times 0 -Scope 'It'
+            }
+        }
+        Context 'Unhappy Paths' {
+            BeforeAll {
+                $encryptionCertificateThumbprint = "some thumbprint"
+            }
+
+            It 'Should create log an error when no secret is returned' {
```
Comment (hckenmiller) [inline Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1]: an error log

File: Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1 [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+                $fabricInstallerSecret = "!!enc!!:some secret"
+                Mock -CommandName  Unprotect-DosInstallerSecret { return $fabricInstallerSecret }
+
+                # Act
+                $decryptedSecret = Get-AuthorizationFabricInstallerSecret `
+                    -fabricInstallerSecret $fabricInstallerSecret `
+                    -encryptionCertificateThumbprint $encryptionCertificateThumbprint
+
+                # Assert
+                $decryptedSecret | Should -eq $fabricInstallerSecret
```
Comment (hckenmiller) [inline Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1]: Shouldn't the decryptedSecret equal "some secret" not "!!enc!!:some secret". I guess it is just a mock test that returns "!!enc!!:some secret"

Comment (ctediz) [reply Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1]: Yea, it's probably better to have the mock return a different value here

File: Fabric.Authorization.API/scripts/Install-Authorization.ps1 (hunk 1 above)
Comment (ctediz) [reply Fabric.Authorization.API/scripts/Install-Authorization.ps1]: I can do that

File: Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1 (hunk 2 above)
Comment (ctediz) [reply Fabric.Authorization.API/scripts/tests/Get-AuthorizationFabricInstallerSecret.tests.ps1]: Will update

**Pull request merged**